.claude/rules/artifacts/observability-placement.md

```
@@ -0,0 +1,168 @@
+### Retention Defaults
+
+| Artifact | Retention | Notes |
+|----------|-----------|-------|
+| Per-step logs | 30 days | Debugging window |
+| Aggregated flow logs | 30 days | Same as step logs |
+| Run-level index | 90 days | Audit trail |
+| Archived logs | Per policy | Cold storage |
+
+## Querying Patterns
+
+### Find All Logs for a Run
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)